extmod/foundation-rust/src/secp256k1.rs

```diff
@@ -87,9 +87,8 @@ pub extern "C" fn secp256k1_sign_schnorr(
             Err(_) => return false,
         };
 
-    let msg = Message::from_digest_slice(data).unwrap();
     let sig =
-        PRE_ALLOCATED_CTX.sign_schnorr_with_rng(&msg, &keypair, &mut rng());
```

Comment:
why does this no longer require a step to get the message from the data?

Reply:
They changed the API to not require a `Message` for Schnorr signatures as that type was only necessary to ensure that when creating ECDSA signatures one used the correct type (32-bytes) as creating a signature from any other data was insecure. This does not happen with Schnorr AFAIK.